containers/self-hosted-runner/test-container.sh

```diff
@@ -0,0 +1,8 @@
+# This command assumes GITHUB_REPOSITORY, GITHUB_REF, and GITHUB_SHA are set on
+# the host.
+
+podman run --rm \
+--env GITHUB_REPOSITORY=${GITHUB_REPOSITORY} \
+--env GITHUB_REF=${GITHUB_REF} \
+--env GITHUB_SHA=${GITHUB_SHA} \
+localhost/fedora-test-runner:latest
```

Comment:
This is a placeholder until this container is pushed to quay.io.

Reply:
does it miss the shebang? 
might want to `exec` the one command